pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -60,6 +66,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.HealthzPort != 0 && utilvalidation.IsValidPortNum(int(kc.HealthzPort)) != nil {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: HealthzPort (--healthz-port) %v must be between 1 and 65535, inclusive", kc.HealthzPort))
 	}
+	if !localFeatureGate.Enabled(features.CPUCFSQuotaPeriod) && kc.CPUCFSQuotaPeriod != zeroDuration && kc.CPUCFSQuotaPeriod != defaultCFSQuota {
```

Comment:
Thinking about this a little more, if 100ms is the default, then why is zeroDuration an exception to the validation?

Reply:
True - that's also covered by the next validation anyways. Will remove that check.